13-spatial-cv.Rmd

```diff
@@ -168,15 +168,16 @@ tm_shape(hs, bbox = bbx) +
 
 
 ## Conventional modeling approach in R {#conventional-model}
-Later on we will introduce the **mlr** package, an umbrella-package providing a unified interface to a plethora of modeling approaches. 
+Later on we will introduce the **mlr** package, an umbrella-package providing a unified interface to a plethora of algorithms. 
 Before doing so, it is worth taking a look at the conventional modeling interface in R.
 This way we introduce statistical supervised modeling in R which provides the required skill set for doing spatial CV and additionally contributes to a better grasp on the **mlr** approach introduced later on.
 Usually, we model the response variable as a function of predictors. 
-Therefore, modeling functions in R such as `lm`, `glm` and many more use the so-called formula interface.
+Therefore, most algorithms implementations in R such as `lm`, `glm` and many more use the so-called formula interface.
 We put this into practice by modeling the landslide occurrence as a function of terrain attributes.
-Since our response (landslide occurrence) belongs to the binary category, we use a binomial generalized linear model instead of a simple linear model which would expect a normally distributed numeric response variable.
+Since our response (landslide occurrence) is binary, we use a generalized linear model with a binomial family and a logit link function instead of a simple linear model which does not allow for other response distributions besides Gaussian.
 
 ```{r, eval = TRUE}
+# meist wird einfach "binomial" geschrieben anstatt binomial()
```

Comment:
yep, this is right but this is what the mlr package is using internally, so I adopted this here (just to show that mlr is doing exactly the same)